scripts/travis_install

```diff
@@ -39,6 +39,9 @@ BK_PKG_NAME=${BK_PKG_NAME/nightly/$VERSION}
 
 conda install --force --yes $BK_PKG_NAME
 
+conda install --yes pip
```

Comment:
now, you have flexx packages, you need to add flexx as the test dependency in the `conda.recipe/meta` file